src/neo/SmartContract/InteropService.Contract.cs

```diff
@@ -166,6 +172,14 @@ private static bool Contract_IsStandard(ApplicationEngine engine)
                 engine.CurrentContext.EvaluationStack.Push(isStandard);
                 return true;
             }
+
+            private static bool Contract_CreateStandardAccount(ApplicationEngine engine)
+            {
+                if (!engine.TryPop(out ReadOnlySpan<byte> pubKey)) return false;
+                UInt160 scriptHash = SmartContract.Contract.CreateSignatureRedeemScript(ECPoint.DecodePoint(pubKey, ECCurve.Secp256r1)).ToScriptHash();
```

Comment:
We decode the point for encode it later. Maybe in the future we should optimize this.

``` public static byte[] CreateSignatureRedeemScript(ECPoint publicKey)
        {
            using (ScriptBuilder sb = new ScriptBuilder())
            {
                sb.EmitPush(publicKey.EncodePoint(true));
                sb.Emit(OpCode.PUSHNULL);
                sb.EmitSysCall(InteropService.Crypto.ECDsaVerify);
                return sb.ToArray();
            }
        }
```

Reply:
This ensures the correctness of the public key format.

But it brought me to the question, will this SYSCALL cause non-deterministic problems? Maybe some implementations will directly encode the incorrectly formatted public key, resulting in inconsistent results of contract execution.

Reply:
We should ensure this with unit tests. And ensure that all implementations pass the same unit tests.

Reply:
@Qiao-Jin could you add a comment for this possible issue in order to warn other implementations ? we must check first that the public key it's valid, before creating the script.

Reply:
> @Qiao-Jin could you add a comment for this possible issue in order to warn other implementations ? we must check first that the public key it's valid, before creating the script.

Have added the comment.